eng/tools/typespec-validation/src/rules/sdk-tspconfig-validation.ts

```diff
@@ -642,6 +642,23 @@ export class TspConfigRustMgmtCrateNameSubRule extends TspconfigEmitterOptionsSu
   }
 }
 
+export class TspConfigRustAzEmitterOutputDirSubRule extends TspconfigEmitterOptionsSubRuleBase {
+  constructor() {
+    super("@azure-tools/typespec-rust", "emitter-output-dir", new RegExp(/{output-dir}\/{service-dir}\/{crate-name}/));
+  }
+
+  protected validate(config: any): RuleResult {
+    let emitterOutputDir = config?.options?.[this.emitterName]?.["emitter-output-dir"];
+    if (emitterOutputDir === undefined) return this.createFailedResult("Failed to find \"options.@azure-tools/typespec-rust.emitter-output-dir\"", "Please add \"options.@azure-tools/typespec-rust.emitter-output-dir\" with a path matching the SDK naming convention \"{output-dir}/{service-dir}/{crate-name}\"");
+    if (emitterOutputDir !== "{output-dir}/{service-dir}/{crate-name}") {
+      return this.createFailedResult(
+        `The value of options.${this.emitterName}.emitter-output-dir "${emitterOutputDir}" does not match the required format "{output-dir}/{service-dir}/{crate-name}"`, "Correct the emitter-output-dir to be \"{output-dir}/{service-dir}/{crate-name}\""
+      );
+    }
```

Comment:
If it's required to be a specific value, kind of begs the question why it's a value at all and not just computed.